install_files/ansible-base/roles/common/tasks/create_users.yml

```diff
@@ -41,8 +44,8 @@
 
 - name: add line to each users bashrc to source the SecureDrop additions
   lineinfile:
-    dest: "/home/{{ item }}/.bashrc"
-    line: ". /etc/bashrc.securedrop_additions"
+    dest: /home/{{ item }}/.bashrc
+    line: '. /etc/bashrc.securedrop_additions'
```

Comment:
Why single quotes here? Double quotes seem to be used almost everywhere else. Do you need the quotes at all?